src/rs.cpp

```diff
@@ -69,17 +70,17 @@ struct rs2_processing_block_list
 
 struct rs2_sensor : public rs2_options
 {
-    rs2_sensor(rs2_device parent,
-        librealsense::sensor_interface* sensor) :
-        rs2_options((librealsense::options_interface*)sensor),
-        parent(parent), sensor(sensor)
+    rs2_sensor( rs2_device parent,
```

Comment:
Please try to avoid unneeded changes, it corrupts the history.
Next time we see a BUG we will say lets see who touch this function last, "oh it's Noa" :)

Reply:
done